src/ui/server/api/keystone/applications/index.ts

```diff
@@ -1,13 +1,18 @@
 import { ApolloResponse } from '../../../../types';
 import apollo from '../../../graphql/apollo';
 import getApplicationsQuery from '../../../graphql/queries/applications';
+import { MAX_APPLICATIONS_PER_PAGE } from '../../../constants';
 
 const applications = {
-  getAll: async (accountId: string) => {
+  getAll: async (accountId: string, skip: number) => {
```

Comment:
could we get a comment to explain what skip is for if anyone else needs to work on this in the future?

Reply:
Good shout, added some documentation 👍